packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1147,14 +1147,14 @@
 	<symbol id="kv-es-section-somethingwentwrong"
```

Comment:
this illustration works with light theme? Probably some colors needs to be defined in a variable.